CodeConverter/Common/ProjectConversion.cs

```diff
@@ -282,7 +282,7 @@ private async Task<WipFileConversion<SyntaxNode>> FirstPassAsync(Document docume
 
     private static string[] GetErrorsFromAnnotations(SyntaxNode convertedNode)
     {
-        var errorAnnotations = convertedNode.GetAnnotations(AnnotationConstants.ConversionErrorAnnotationKind).ToList();
+        var errorAnnotations = convertedNode.DescendantNodesAndSelf().SelectMany(t => t.GetAnnotations(AnnotationConstants.ConversionErrorAnnotationKind)).ToList();
```

Comment:
Yeah looks accidental, I probably thought it would recurse since I think [`GetAnnotatedNodes`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.syntaxnode.getannotatednodes?view=roslyn-dotnet-4.2.0 does) must do